packages/cli/README.md

```diff
@@ -14,150 +14,208 @@ Run the following command to install the CLI.
+            --skip-install   # Do not automatically install dependencies  Default: false
+            --description    # Description for the extension
+            --outdir         # Project root directory for the extension
+            --tslint         # Enable tslint
+            --prettier       # Enable prettier
+            --mocha          # Enable mocha
+            --loopbackBuild  # Use @loopback/build
+            --componentName  # Component name
+    ```
+
+3.  To scaffold a Controller into your application
+
+    ```sh
+    cd <your-project-directory>
+    lb4 controller
+    ```
+
+    ```sh
+    Usage:
+      lb4 controller [options] [<name>]
+
+    Options:
+      -h,   --help            # Print the generator's options and usage
+            --skip-cache      # Do not remember prompt answers             Default: false
+            --skip-install    # Do not automatically install dependencies  Default: false
+            --controllerType  # Type for the controller
+
+    Arguments:
+      name  # Name for the controller  Type: String  Required: false
+    ```
 
 4.  To scaffold a DataSource into your application
 
     `lb4 datasource`
 
-```sh
-Usage:
-  lb4 datasource [options] [<name>]
+    ```sh
+    Usage:
+      lb4 datasource [options] [<name>]
 
-Options:
-  -h,   --help            # Print the generator's options and usage
-        --connector       # Name of datasource connector
+    Options:
+      -h,   --help            # Print the generator's options and usage
+            --connector       # Name of datasource connector
 
-Arguments:
-  name  # Name for the datasource  Type: String  Required: true
-```
+    Arguments:
+      name  # Name for the datasource  Type: String  Required: true
+    ```
 
 5.  To scaffold a Model into your application
 
     `lb4 model`
 
-```sh
-Usage:
-  lb4 model [options] [<name>]
+    ```sh
+    Usage:
+      lb4 model [options] [<name>]
 
-Options:
-  -h,   --help            # Print the generator's options and usage
+    Options:
+      -h,   --help            # Print the generator's options and usage
+            --base            # A valid base model
 
-Arguments:
-  name  # Name for the model  Type: String  Required: true
-```
+    Arguments:
+      name  # Name for the model  Type: String  Required: true
+    ```
 
-6.  To download one of LoopBack example projects
+6.  To scaffold a Repository into your application
+
+    `lb4 repository`
+
+    ```sh
+    Usage:
+      lb4 repository [options] [<name>]
+
+    Options:
+      -h,   --help           # Print the generator's options and usage
+            --model          # A valid model name
+            --id             # A valid ID property name for the specified model
+            --datasource     # A valid datasource name
+
+    Arguments:
+      name  # Name for the repository   Type: String  Required: false
+    ```
+
+7.  To scaffold a Service into your application
+
+    `lb4 service`
```

Comment:
Nitpick: I like the hint of going into the project directory in the doc for `lb4 controller`, could you add it for the new commends as well?

```
    cd <your-project-directory>
```

Same comment apply to the other commends.